python/ml4ir/base/config/eval_config.py

```diff
@@ -0,0 +1,55 @@
+from ml4ir.applications.ranking.model.metrics.helpers.metrics_helper import RankingConstants
+
+class EvalConfigConstants:
```

Comment:
@mohazahran Do you not need a notion of "higher is better" or "lower is better" to compute the p values for the metrics?

Reply:
no, I'm using two sided test